build_tools/update_environments_and_lock_files.py

```diff
@@ -90,6 +90,32 @@ def remove_from(alist, to_remove):
 
 
 build_metadata_list = [
+    {
+        "name": "pylatest_conda_forge_mkl_array-api_linux-64",
+        "type": "conda",
+        "tag": "main-ci",
```

Comment:
It should probably be its main tag like `"gpu"` or something.

Do you want the lock-file to be updated every week?

Reply:
Let's update the lockfile. I think it would be weird to be different from the other envs.

Updated the tag.